WireAPI/Package.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version: 5.10
+// swift-tools-version: 6.0
```

Comment:
Is setting `6.0` an important part of the fix? I think in many cases this will break migration to Xcode 16 - have you tested this?

Reply:
not sure if it was needed, i reverted the change, I am checking if the CI can build first like this. to my understanding that comes from develop, no?

Reply:
at the end do we need separate package files ?

Reply:
I assume it is coming from develop, much of it from my PR - https://github.com/wireapp/wire-ios/pull/1869/files which was largely changing `6.0` to ` 5.10` and deleting the `Package@swift-5.10.swift` files